README.md

```diff
@@ -1,2 +1,222 @@
-# credit-card_defaults_prediction
+# credit_card_defaults_prediction
 Repository to Collaborate on Capstone project
+
+## Cloning this repository:
+
+If you would like to clone this repository and actively develop new features for this, here are the steps to clone this to your local machine and get started.
+
+```bash
+$ cd <some>/<dir>/
+$ git clone https://github.com/amarchini5339/credit-card_defaults_prediction.git
+$ cd ./credit-card_defaults_prediction
+```
+
+## Prerequisites: 
+
+### Python Version
+For this project we are using Python version 3.10.15, conda automatically will install and set the correct python version for the project so there is nothing that needs to be done.
+
+### 1. Install Miniconda
```

Comment:
If you already have anaconda installed, you don't need to install miniconda. In fact, it could create issues. @DatasanAli

Reply:
That's a good point, didn't think about that. Yea if you have anaconda you don't need to install miniconda. Nice catch. I'll update the readme to reflect that.